actor/builtin/miner/sector_set.go

```diff
@@ -0,0 +1,74 @@
+package miner
+
+import (
+	"strconv"
+
+	"github.com/filecoin-project/go-filecoin/types"
+	"github.com/filecoin-project/go-filecoin/vm/errors"
+)
+
+// Note that this type's interface is a bit of a placeholder.  We expect to
+// move to something like actor.Lookup eventually which will manage its own
+// hash linked storage.  See #2887.  This will require some interface changes.
+
+// SectorSet is a collection of sector commitments indexed by their integer
+// sectorID.  Due to a bug in refmt, the sector id-keys need to be stringified.
+// See also: https://github.com/polydawn/refmt/issues/35.
+type SectorSet map[string]types.Commitments
```

Comment:
This type decouples the SectorSet's implementation from its interface a bit better than before which helps with updating the miner actor's commitments and proving sets.  This type doesn't capture the interface we can expect in the near future which will need to pass around the miner's storage and return cids.